pkg/operation/cloudbotanist/alicloudbotanist/infrastructure.go

```diff
@@ -153,6 +173,63 @@ func (b *AlicloudBotanist) generateTerraformBackupConfig() map[string]interface{
 		"bucket": map[string]interface{}{
 			"name": b.Operation.BackupInfrastructure.Name,
 		},
-		"clusterName": b.Operation.BackupInfrastructure.Name,
 	}
 }
+
+func cleanSnapshots(bucketName, storageEndPoint, accessKeyID, accessKeySecret string) error {
+	client, err := generateOSSClient(storageEndPoint, accessKeyID, accessKeySecret)
+	if err != nil {
+		return err
+	}
+	bucket, err := client.Bucket(bucketName)
+	if err != nil {
+		return err
+	}
+	snapshots, err := listSnapshot(bucket)
+	if err != nil {
+		return err
+	}
+	err = deleteObjects(bucket, snapshots)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func generateOSSClient(storageEndpoint, accessKeyID, accessKeySecret string) (*oss.Client, error) {
+	client, err := oss.New(storageEndpoint, accessKeyID, accessKeySecret)
```

Comment:
Remove this function - it is of no help (instead of `oss.New(...)` you now call `generateOSSClient(...)` with exactly the same arguments and no different behaviour.